src/components/ModalWithForm.js

```diff
@@ -9,23 +9,27 @@ class ModalWithForm extends Modal {
   }
 
   close() {
-    this._modalForm.reset();
     super.close();
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`